hypothesis-python/src/hypothesis/provisional.py

```diff
@@ -31,6 +31,32 @@
 import hypothesis._strategies as st
 
 
+@st.defines_strategy_with_reusable_values
+def urls():
+    """"A strategy for :rfc:`3986`, generating http/https URLs."""
```

Comment:
Oh, and apparently [`flake8` doesn't like the four quotes to start this](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/476086938#L806).  It took me a long time to see that there were four instead of three :man_facepalming: